### ISSUE TITLE
Question about Copyright notation

### ISSUE THREAD

**Issue by masahir0y** [ARM-software/tf-issues#471]
I'd like to be sure about the Copyright notation before I start
to upstream Socionext SoC code.

I looked into plat/ directory, but I did not see any vendor-specific copyright
(such as NVIDIA, Xilinx, etc.)

The header part is always like this:
Copyright (c) 2016-2017, ARM Limited and Contributors. All rights reserved.

This is the preferred notation throughout the ARM Trusted Firmware repository.  Right?
In other words, SoC vendors are included in the term "and Contributors".

**Comment by danh-arm** [comment]
Yes, that is the preferred copyright notation in the codebase to avoid unnecessary proliferation of copyright notices. The only deviation is when we import OSS code into the project without the permission of the copyright owner (i.e. the license obliges us to retain the existing notices).

You can add Socionext to the list of contributors in the [acknowledgments file](https://github.com/ARM-software/arm-trusted-firmware/blob/master/acknowledgements.md) if you like, but this is optional.
